test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_Java8.xml

```diff
@@ -513,12 +513,32 @@
   <output regex="no" type="success">arg1 = \u0058</output>
   <output regex="no" type="success">arg2 = \\u0058</output>
   </test>
-  
+
  <test id="Test system properties that resemble Unicode escapes (with translation)">
   <command>$EXE$ -Xargencoding -Darg1=\u0058 -Darg2=\\u0058 -XshowSettings:properties -version</command>
   <output regex="no" type="success">arg1 = X</output>
   <output regex="no" type="success">arg2 = \X</output>
   </test>
-  
+
+ <test id="Test option -Xjit:count=0,disableZ196">
+  <command>$EXE$ -Xjit:count=0,disableZ196 -version</command>
+  <output regex="no" type="success">version</output>
+ </test>
+
+ <test id="Test option -Xjit:count=0,disableZEC12">
+  <command>$EXE$ -Xjit:count=0,disableZEC12 -version</command>
+  <output regex="no" type="success">version</output>
+ </test>
+
+ <test id="Test option -Xjit:count=0,disableZ13">
+  <command>$EXE$ -Xjit:count=0,disableZ13 -version</command>
+  <output regex="no" type="success">version</output>
+ </test>
+
+ <test id="Test option -Xjit:count=0,disableZ14">
+  <command>$EXE$ -Xjit:count=0,disableZ14 -version</command>
+  <output regex="no" type="success">version</output>
+ </test>
```

Comment:
Let's also add a `disableZ15` test as well.

Reply:
Added in 0a6890d